insights/parsers/tests/test_freeipa_healthcheck_log.py

```diff
@@ -0,0 +1,18 @@
+from insights.parsers.freeipa_healthcheck_log import FreeIPAHealthCheckLog
+from insights.tests import context_wrap
+
+FREEIPA_HEALTHCHECK_LOG_OK = """[{"source": "ipahealthcheck.meta.services", "check": "certmonger", "severity": 0, "uuid": "693c526c-2261-44fe-98e0-94c6b76bf79b", "when": "20190711141703Z", "duration": "0.012848", "kw": {"status": true}}]"""
```

Comment:
When you use long strings, an alternative way to define them is something like the following:
```
LONG_STR = """
a very long string
 with lots of words
 that can be hard to read
""".strip()

long_str = "".join(LONG_STR.splitlines())
```
Just make sure you have space at the end or beginning of the line so that separate words are not mashed together.

Reply:
Updated, thank you.